src/test/java/core/basesyntax/service/impl/TransactionProcessorImplTest.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.TransactionProcessor;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class TransactionProcessorImplTest {
+    private static TransactionProcessor transactionProcessor;
+
+    @BeforeClass
+    public static void init() {
+        transactionProcessor = new TransactionProcessorImpl();
+    }
+
+    @Test
+    public void process_validTransactions_ok() {
+        List<FruitTransaction> fruitTransactions = List.of(
+                new FruitTransaction(FruitTransaction.Operation.BALANCE, "apple", 20),
+                new FruitTransaction(FruitTransaction.Operation.PURCHASE, "apple", 7));
+        transactionProcessor.process(fruitTransactions);
+    }
+
+    @Test
+    public void process_emptyList_ok() {
+        List<FruitTransaction> fruitTransactions = new ArrayList<>();
+        transactionProcessor.process(fruitTransactions);
+    }
```

Comment:
You're just calling method here, but don't check anything. You should add assertEquals in the end of each test, where your actual value will be read from the storage after processing.

Reply:
add asserts